exampleCourse/questions/element/details/question.html

```diff
@@ -0,0 +1,19 @@
+<pl-question-panel>
+  <p>
+    The below element uses the pl-details component to show and hide content. 
+    The pl-details component is shown by default, and the content is hidden when you click on the title.
+    The pl-details component can be used to show or hide any content, including other components.
+  </p>
+
+  <pl-details title="Additional information">
+    <p>For this question, you should click on the next header labeled "Large table of data" to view the content.</p>
+  </pl-details>
+
+  <p>
+    The below element has set <code>show="false"</code> to hide the content by default.
+    When you click on the title, the content will be shown.
+  </p>
+  <pl-details title="Large table of data" show="false">
```

Comment:
In the interested of illustrating best practices:

- I don't think the word "large" needs to be here.
- It'd be better if we could actually label something about the data ("Breast cancer treatment data" or whatever)

Reply:
Changed in [f81b596](https://github.com/PrairieLearn/PrairieLearn/pull/11929/commits/f81b59665f57416b3f4efda0af559efa039fcf2c).